src/react/existing-search-params.tsx

```diff
@@ -48,6 +51,7 @@ export function ExistingSearchParams({ exclude }: Props) {
 						type="hidden"
 						name={key}
 						value={value}
+						{...props}
```

Comment:
Let’s move this to be before type, that way someone using JS can’t overwrite the fixed props.

Reply:
Makes sense, updated